pkg/routeagent_driver/main.go

```diff
@@ -152,3 +155,23 @@ func annotateNode(clusterCidr []string, cfg *restclient.Config) error {
 
 	return nil
 }
+
+func getUsrForceMssVal(cfg *restclient.Config) int {
+	usrForceMssVal := 0
+
+	k8sClientSet, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		klog.Errorf("Error building clientset: %s", err.Error())
+		return usrForceMssVal
+	}
+
+	localNode, err := node.GetLocalNode(k8sClientSet)
+	if err != nil {
+		klog.Errorf("Error  getting information on the local node: %s", err.Error())
+		return usrForceMssVal
+	}
+
+	usrForceMssVal, _ = strconv.Atoi(localNode.GetAnnotations()[v1.ForceTCPMssClamping])
```

Comment:
Since this is an user input, we should validate if there is any error in this conversion.